Source/WTF/wtf/FastMalloc.h

```diff
@@ -684,7 +684,7 @@ void operator delete(T* object, std::destroying_delete_t, size_t size) { \
     ASSERT(sizeof(T) == size); \
     object->T::~T(); \
     if (UNLIKELY(object->checkedPtrCountWithoutThreadCheck())) { \
-        memset(static_cast<void*>(object), 0, size); \
+        zeroSpan(unsafeMakeSpan(reinterpret_cast<char*>(object), size)); \
```

Comment:
We should be able to use `zeroBytes(object);` since `sizeof(object) == size` per the `static_assert` above.